### PR TITLE
fix rp server failed to collect pprof error

### PR DESCRIPTION
**Pull request by sonyafenge**
**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:
rp server failed to collect pprof becuase there is no kube-scheduler running. Add component running judgment, only collecting pprof for running component.
**Which issue(s) this PR fixes**:

Fixes #https://github.com/CentaurusInfra/arktos/issues/1057

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cluster/gce/gci/configure-helper-common.sh [hunk 1]
```diff
@@ -1467,79 +1467,28 @@ function start-collect-pprof {
     COMPONENTS_PORTS="8080"
     COMPONENTS_NAME="kube-apiserver"
     echo "Collecting kube-apiserver pprof at ${CURRENTTIME}"
-    collecting-pprof ${COMPONENTS_PORTS} ${COMPONENTS_NAME} 
+    ps -C kube-apiserver >/dev/null && collecting-pprof ${COMPONENTS_PORTS} ${COMPONENTS_NAME} || echo "kube-apiserver is not running on this machine"
```
Comment (Sindica) [inline cluster/gce/gci/configure-helper-common.sh:1470]: Somehow I feel line 1467, 1468, 1470 are overprogrammed :)

Comment (sonyafenge) [reply cluster/gce/gci/configure-helper-common.sh:1470]: how about directly use?
```
ps -C kube-apiserver >/dev/null && collecting-pprof "8080" "kube-apiserver" || echo "kube-apiserver is not running on this machine"
```

Comment (Sindica) [reply cluster/gce/gci/configure-helper-common.sh:1470]: > how about directly use?
> 
> ```
> ps -C kube-apiserver >/dev/null && collecting-pprof "8080" "kube-apiserver" || echo "kube-apiserver is not running on this machine"
> ```

I like this better :)

**Review by Sindica — APPROVED** [review]
/lgtm

**Review by XiaoningDing — APPROVED** [review]

**Pull request merged**